target/mips/op_helper.c

```diff
@@ -2823,6 +2833,7 @@ static void r4k_dump_tlb(CPUMIPSState *env, int idx)
 #if 1
```

Comment:
Unrelated but shouldn't this be TARGET_CHERI?

Reply:
I was confused, too, since the whole block is in an `ifdef TARGET_CHERI` already (see just above r4k_dump_tlb), but I decided to follow along.